### PR TITLE
update of outputs for walkthrough

### PR DESCRIPTION
**Pull request by drnic**
This PR is a 
 - [ ] Feature Implementation
 - [ ] Bug Fix
 - [x] Documentation

**What this PR does / why we need it**:

When I did the walkthrough for deploying the ups-broker I noticed the output had changed slightly since the documentation was written. I've updated the example output; and cleaned up some markdown.

### CODE REVIEW

File: docs/walkthrough.md [hunk 1]
```diff
@@ -106,15 +106,17 @@ provides. We can view the `ClusterServiceClass` resources available:
...
-+-----------------------------------+-------------------------+--------------------------------------+
-  user-provided-service               A user provided service   4f6e6cf6-ffdd-425f-a2c7-3c9258ad2468
-  user-provided-service-single-plan   A user provided service   5f6e6cf6-ffdd-425f-a2c7-3c9258ad2468
+                 NAME                  NAMESPACE         DESCRIPTION
++------------------------------------+-----------+-------------------------+
+  user-provided-service                            A user provided service
+  user-provided-service-single-plan                A user provided service
+  user-provided-service-with-schemas               A user provided service
 
 $ kubectl get clusterserviceclasses -o=custom-columns=NAME:.metadata.name,EXTERNAL\ NAME:.spec.externalName
```
Comment (luksa) [inline docs/walkthrough.md]: There's no longer a need to specify the custom output format. You should now be able to simply run `kubectl get clusterserviceclasses` and get the same nice output.

File: docs/walkthrough.md [hunk 2]
```diff
@@ -15,23 +15,23 @@ of this repository.
...
+proceeding, and we'll do so with the UPS helm chart. You can find details about
+that chart in the chart's
 [README](https://github.com/kubernetes-incubator/service-catalog/blob/master/charts/ups-broker/README.md).
 
 Otherwise, to install with sensible defaults, run the following command:
 
-```
-helm install charts/ups-broker --name ups-broker --namespace ups-broker
+```console
+helm install svc-cat/ups-broker --name ups-broker --namespace ups-broker
```
Comment (luksa) [inline docs/walkthrough.md]: Why this change? Aren't the charts in the `charts/` folder any more?

File: docs/walkthrough.md [hunk 3]
```diff
@@ -165,40 +175,52 @@ resources available in the cluster:
...
-  default   user-provided-service-single-plan   Sample plan description   96064792-7ea2-467b-af93-ac9694d96d52
+   NAME     NAMESPACE                 CLASS                           DESCRIPTION
++---------+-----------+------------------------------------+--------------------------------+
+  default               user-provided-service                Sample plan description
+  premium               user-provided-service                Premium plan
+  default               user-provided-service-single-plan    Sample plan description
+  default               user-provided-service-with-schemas   Plan with parameter and
+                                                             response schemas
 
 $ kubectl get clusterserviceplans -o=custom-columns=NAME:.metadata.name,EXTERNAL\ NAME:.spec.externalName
```
Comment (luksa) [inline docs/walkthrough.md]: As above: don't need `-o` any more.

**Comment by drnic** [comment]
I've now signed the CLA on behalf of Stark & Wayne.

File: docs/walkthrough.md (hunk 2 above)
Comment (drnic) [reply docs/walkthrough.md]: @luksa Ah this command sensitive to where you run it; ok. Can I change this to `helm install ./charts/ups-broker...` so its clear its a local path rather than referencing the hosted `svc-cat/ups-broker` chart?

File: docs/walkthrough.md (hunk 1 above)
Comment (drnic) [reply docs/walkthrough.md]: Ok, thanks, fixed.

**Comment by drnic** [comment]
Updated from @luksa commands and rebased against master

**Comment by jboyd01** [comment]
Thanks for making time to update this @drnic.
/approve
